readme.md

```diff
@@ -1,6 +1,6 @@
 # Hibernate OGM
 
-Version: 5.0.0.Final - 13 May 2016
+Version: 5.1.0-SNAPSHOT
```

Comment:
I don't like that the README on github advertises the final version when it's not.

+1 to remember updating it during release, I always do on Search as one has to set the date.